main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -444,6 +464,26 @@ object ParsedAst {
       */
     case class QName(sp1: SourcePosition, name: Name.QName, sp2: SourcePosition) extends ParsedAst.Expression
 
+    /**
+      * Simply Qualified Name Expression: The qualifier is either or a class or a top-level namespace.
+      *
+      * @param sp1 the position of the first character in the expression.
+      * @param qualifier the namespace or class.
+      * @param name the name.
+      * @param sp2 the position of the last character in the expression.
+      */
+    case class SQName(sp1: SourcePosition, qualifier: Name.Ident, name: Name.Ident, sp2: SourcePosition) extends ParsedAst.Expression
+
+    /**
+      * Qualified Sig Expression
+      *
+      * @param sp1   the position of the first character in the expression.
+      * @param clazz  the name of the class.
+      * @param sig the name of the signature.
+      * @param sp2   the position of the last character in the expression.
+      */
+    case class QSig(sp1: SourcePosition, clazz: Name.QName, sig: Name.Ident, sp2: SourcePosition) extends ParsedAst.Expression
```

Comment:
clazz -> qname

Reply:
I mean, our naming is not very precise. Maybe we should fix that in the future. On the other hand, I also have no desire for super long names like in Java.